test/connectObservable.test.tsx

```diff
@@ -136,47 +198,52 @@ describe("connectObservable", () => {
 
     render(<TestSuspense />)
 
-    expect(screen.queryByText("Result")).toBeNull()
-    expect(screen.queryByText("Waiting")).not.toBeNull()
+    expect(screen.queryByText("Result 0")).not.toBeNull()
+    expect(screen.queryByText("Waiting")).toBeNull()
 
     fireEvent.click(screen.getByText(/Next/i))
 
-    expect(screen.queryByText("Result")).toBeNull()
     expect(screen.queryByText("Waiting")).not.toBeNull()
 
     await wait(110)
 
     expect(screen.queryByText("Result 1")).not.toBeNull()
     expect(screen.queryByText("Waiting")).toBeNull()
+  })
 
-    fireEvent.click(screen.getByText(/Next/i))
-
-    expect(screen.queryByText("Result")).toBeNull()
-    expect(screen.queryByText("Waiting")).not.toBeNull()
+  it("keeps in suspense if more than two SUSPENSE are emitted in succesion", async () => {
```

Comment:
> I guess it doesn't hurt for it to emit more than one (or does it?)

I just added this test and it verifies that it doesn't hurt - it doesn't break suspense.